zeg/imagesets.py

```diff
@@ -410,9 +410,29 @@ def _resolve_paths(paths, should_recursive, ignore_mime, log):
                 )
         elif os.path.isfile(path) and whitelisted:
             resolved.append(path)
+
+    total_size = 0
+    warned = False
+    for path in resolved:
+        size = os.path.getsize(path)
+        total_size += size
+        if size > 268435456 and not warned:
```

Comment:
probably a better place to put this limit, maybe some config somewhere. Not urgent though

Reply:
I moved it to the top near the mime types and whitelist vars.